api/operations/modify_group_users.py

```diff
@@ -369,8 +370,25 @@ async def _execute(self) -> OktaGroup:
                                 )
                             )
 
-        # Commit all changes so far
-        db.session.commit()
+            db.session.commit()
+
+            # Invoke app group lifecycle plugin hooks for removed members
+            plugin_id = get_app_group_lifecycle_plugin_to_invoke(self.group)
+            if plugin_id is not None and len(self.members_to_remove) > 0:
+                try:
+                    hook = get_app_group_lifecycle_hook()
+                    hook.group_members_removed(
+                        session=db.session, group=self.group, members=self.members_to_remove, plugin_id=plugin_id
+                    )
+                    db.session.commit()
+                except Exception:
+                    current_app.logger.exception(
+                        f"Failed to invoke group_members_removed hook for group {self.group.id} with plugin '{plugin_id}'"
+                    )
+                    db.session.rollback()
+        else:
+            # Commit all changes so far
+            db.session.commit()
```

Comment:
This session commit probably doesn't need to be in an else clause, but either way is fine for now.